mounts_docker.go

```diff
@@ -0,0 +1,116 @@
+package testcontainers
+
+import "github.com/docker/docker/api/types/mount"
+
+var (
+	mountTypeMapping = map[MountType]mount.Type{
+		MountTypeBind:   mount.TypeBind,
+		MountTypeVolume: mount.TypeVolume,
+		MountTypeTmpfs:  mount.TypeTmpfs,
+		MountTypePipe:   mount.TypeNamedPipe,
```

Comment:
I cannot see an implementation for `MountTypePipe` but there are for Bind, Volume and Tmpfs. Is this on purpose?